scripts/log-serial/log-serial.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Serial Logger Service
+
+[Service]
+ExecStart=/usr/local/bin/log-serial.py /var/log/afcv4-openmmc-serial.log
```

Comment:
It would be interesting to allow setting the serial port path when launching the service similarly the way we do for PCIe slots when launching the afc-ioc: [https://github.com/lnls-dig/afc-epics-ioc/blob/master/service/afc-ioc%40.service](https://github.com/lnls-dig/afc-epics-ioc/blob/master/service/afc-ioc%40.service)